.circleci/config.yml

```diff
@@ -48,114 +52,80 @@ jobs: # a collection of steps
-          command: ./backend/node_modules/.bin/lcov-result-merger './backend/reports/coverage/lco*.info' './backend/reports/coverage/lcov_merged.info'
-      - run:
-          command: cat ./backend/reports/coverage/lcov_merged.info | ./backend/node_modules/.bin/coveralls
-          environment:
-            COVERALLS_SERVICE_NAME: circle-ci
-            COVERALLS_GIT_BRANCH: ${CIRCLE_BRANCH}  
-      - run:
-          name: Define environment variable with lastest commit's message
-          command: |
-            echo 'export COMMIT_MESSAGE=$(git log -1 --pretty=format:"%s")' >> $BASH_ENV
-            source $BASH_ENV
+      - run: yarn --pure-lockfile
+      - run: yarn run ci
+      - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+      # To help debug when lerna detects changes to the working tree and fails the publish
+      - run: git status
+      - run: yarn run release:publish
+
+  prepare-vsix:
+    docker:
+      - image: circleci/node:14
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: yarn
+      - run: yarn run ci
+      - run: mkdir -p artifacts && cp packages/*/*.vsix ./artifacts
       - run:
-          name: Lint commit message
-          working_directory: backend
-          command: echo "$COMMIT_MESSAGE" | npx commitlint     
-      - save_cache:
-          paths:
-            - backend/node_modules
-            - frontend/node_modules
-          key: dependency-cache-{{ checksum "backend/package.json" }}-{{ checksum "frontend/package.json" }}
+          name: "Inspect Artifacts Folder"
+          working_directory: ./artifacts
+          command: ls -la
       - persist_to_workspace:
-          root: backend
-          paths: 
-           # https://golang.org/pkg/path/filepath/#Match
-            - yeoman-ui-*
-      - store_test_results: # for display in Test Summary: https://circleci.com/docs/2.0/collect-test-data/
-          path: test-results.xml
-  
-  deploy:
+          root: "./artifacts"
+          paths:
+            # https://golang.org/pkg/path/filepath/#Match
+            - "*.vsix"
+
+  deploy-gh-releases:
     docker:
       - image: circleci/golang:1.13
     steps:
       - attach_workspace:
-          at: ./dist
+          at: ./artifacts
+      - run:
+          name: "Inspect Artifacts Folder"
+          working_directory: ./artifacts
+          command: ls -la
       - run:
           name: "Publish Release on GitHub"
           command: |
             go get github.com/tcnksm/ghr
-            ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} ${CIRCLE_TAG} ./dist/ 
+            VERSION=${CIRCLE_TAG}
+            ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ./artifacts/
 
 workflows:
   version: 2
-  test-deploy:
+  voter:
     jobs:
-      - build-and-test-node10
-      - build-and-test-node12
+      - build-node10
```

Comment:
node10 is EOL in 3 weeks

Reply:
@tomer-epstein, @rimasirich, may I remove node 10 build?